source/partials/next_release/_release.md.erb

```diff
@@ -36,6 +42,7 @@ Users can also utilize the newly introduced `branch` attribute on Mercurial mate
 
 * <%= deprecated_api_message api_name: 'Pipeline config', old_api_version: 6, new_api_version: 7, new_api_available_in_release: '19.4.0', api_section: 'pipeline-config', removal_date_and_year: 'Aug 2019' %>
 * <%= deprecated_api_message api_name: 'Config repo', old_api_version: 1, new_api_version: 2, new_api_available_in_release: '19.4.0', api_section: 'config-repo', removal_date_and_year: 'Aug 2019' %>
+* The support for Java 8 has been deprecated.
```

Comment:
Support for Java 8 has been deprecated. The support for the same will be removed in a release scheduled for November 2018